po/uk.po

```diff
@@ -92,7 +92,8 @@ msgstr "ІД КЛЮЧА"
 #: app/flatpak-builtins-build-sign.c:44
 #: app/flatpak-builtins-build-update-repo.c:84
 msgid "GPG Homedir to use when looking for keyrings"
-msgstr "Домашній каталог GPG, яким слід користуватися для пошуку сховищ ключів"
+msgstr ""
```

Comment:
Is there a reason for the empty string here? Looks like the actual translation is on the next line

Reply:
This appears to also be the case for other translations in this file.